README.md

```diff
@@ -34,9 +34,11 @@ See [CHANGELOG.md](CHANGELOG.md) for the latest updates.
 - [Microsoft Copilot](https://github.com/Tolga1452/ai-prompts/tree/main/Microsoft%20Copilot)
 - [v0](https://github.com/Tolga1452/ai-prompts/tree/main/v0)
 - [Visual Studio Code](https://github.com/Tolga1452/ai-prompts/tree/main/Visual%20Studio%20Code)
+- [Bluesky Bot](https://github.com/Tolga1452/ai-prompts/tree/main/Bluesky%20Bot)
```

Comment:
```suggestion
```

All the categories are in the alphabet order. Can you please move it under "Apple Intelligence"?